frontend/src/lib/stores.js

```diff
@@ -1,20 +1,49 @@
-import words from '../languages/et/words.json'
 import { writable } from 'svelte/store'
+import languages from '../languages/languages.json'
+
+let wordCache
+let cachedLanguage
+
+async function getWords() {
```

Comment:
Just one last thing, I'd move this function to be above `randInt` so the `_currentLanguage` stuff is defined above it and it isn't as unclear

Reply:
ok